app/views/layouts/_flash.html.erb

```diff
@@ -1,9 +1,13 @@
 <div class="grid-row">
   <div class="column-two-thirds">
-    <% flash.each do |key, value| %>
-      <div class="govuk-<%= key %>-summary" role="alert" aria-labelledby="<%= key %>-summary-heading" tabindex="-1" data-module="error-summary">
-        <h3 class="govuk-<%= key %>-summary__title" id="<%= key %>-summary-heading"><%= value %></h3>
-      </div>
+    <% if flash[:publish_errors].any?%>
+      <%= render partial: 'layouts/publish_errors' %>
```

Comment:
We could make this generic as I'm sure we'll re-use it and call it `error_summary`